app/src/main/res/layout/list_item_favorite.xml

```diff
@@ -45,47 +44,47 @@
 			android:layout_marginTop="8dp"
 			android:ellipsize="middle"
 			android:singleLine="true"
-			android:textColor="?android:textColorPrimary"
 			app:layout_constraintLeft_toRightOf="@+id/fromIcon"
 			app:layout_constraintRight_toLeftOf="@+id/overflowButton"
 			app:layout_constraintTop_toTopOf="parent"
-			tools:text="From Station that can have a very long name"/>
+			tools:text="From Station that can have a very long name"
+			style="@style/FavoriteCardForeground" />
 
 		<ImageView
 			android:id="@+id/viaIcon"
 			android:layout_width="wrap_content"
 			android:layout_height="0dp"
 			android:scaleType="centerCrop"
-			app:tint="?colorControlNormal"
 			app:layout_constraintBottom_toBottomOf="@+id/via"
 			app:layout_constraintLeft_toLeftOf="@+id/fromIcon"
 			app:layout_constraintTop_toTopOf="@+id/via"
 			app:srcCompat="@drawable/trip_via"
 			tools:ignore="ContentDescription"
-			tools:visibility="visible"/>
+			tools:visibility="visible"
+			style="@style/FavoriteCardForeground" />
 
 		<TextView
 			android:id="@+id/via"
 			android:layout_width="0dp"
 			android:layout_height="wrap_content"
 			android:ellipsize="middle"
 			android:singleLine="true"
-			android:textColor="?android:textColorPrimary"
```

Comment:
As said before, not a fan of having Material You-adapted text colors.

Reply:
This is different than the above one. Here we actually re-set the text color via the FavoriteCardForeground-style. That sets  the text color. Removing the coloring makes it look like this:

![image](https://github.com/grote/Transportr/assets/25279821/aa496c0f-48f1-4696-a253-47769f6f54c7)
We will have to set the text color anyway to fix the tinting. What color would you like?